substrafl/remote/register/register.py

```diff
@@ -41,19 +50,22 @@
 # install dependencies
 RUN python{python_version} -m pip install -U pip
 
-# Install substrafl, substra (and substratools if editable mode)
-{cl_deps}
-
-# PyPi dependencies
-{pypi_dependencies}
+# Copy local wheels
+{copy_wheels}
```

Comment:
So will there still be a "substrafl_internal folder" or everything will be at root ?